index.js

```diff
@@ -45,9 +45,11 @@ function PinoColada () {
 
     if (!obj.level) obj.level = 'userlvl'
     if (!obj.name) obj.name = ''
+    if (!obj.ns) obj.ns = ''
```

Comment:
do you think we should just display one or the other? i.e. is there an instance where we are going to have both `obj.ns` and `obj.name` ?